lib/routes/account.js

```diff
@@ -484,13 +501,17 @@ module.exports = function (
                   uid: emailRecord.uid,
                   kA: emailRecord.kA,
                   wrapKb: wrapKb,
-                  emailVerified: emailRecord.emailVerified
+                  emailVerified: emailRecord.emailVerified,
+                  tokenVerificationId: tokenVerificationId,
+                  tokenVerified: false
                 })
               }
             )
             .then(
               function (keyFetchToken) {
                 response.keyFetchToken = keyFetchToken.data.toString('hex')
+                response.challengeMethod = 'email';
+                response.challengeReason = 'signin';
```

Comment:
Also, if `challengeReason` and `challengeMethod` are set, `response.verified` must be set to `false` to inform existing clients like Fx Desktop to poll `/recovery_email/status` without knowing to check the new fields.